terraform/router/nginx/main.tf

```diff
@@ -1,12 +1,13 @@
 resource "kubernetes_config_map" "nginx-configuration" {
+  count = var.cloud_provider == "aws" ? 0 : 1
```

Comment:
what is this variable used for?

Reply:
0 means the resource won't be created and 1 it will be created.
In the case of aws we are now managing the configmap on the aws module.